indico/web/forms/jinja_helpers.py

```diff
@@ -16,13 +16,13 @@
                                      IndicoSelectMultipleCheckboxField)
 from indico.web.forms.validators import (ConfirmPassword, HiddenUnless, IndicoRegexp, SecurePassword, SoftLength,
                                          WordCount)
-from indico.web.forms.widgets import RemoteDropdownWidget, TypeaheadWidget
+from indico.web.forms.widgets import DropdownWidget, TypeaheadWidget
 
 
 def is_single_line_field(field):
-    if isinstance(field.widget, (RemoteDropdownWidget, TypeaheadWidget)):
+    if isinstance(field.widget, TypeaheadWidget):
         return True
-    if isinstance(field.widget, Select):
+    if isinstance(field.widget, (DropdownWidget, Select)):
```

Comment:
Are you sure this is correct? We use `Select` here because it becomes bigger in `multiple` mode. But the SUI Dropdown has the same size regardless of whether it's for a single value or multiple values. So I think it should stay above in the `return True` case.

Reply:
Well I guess it doesn't make much difference: 
This is the position of the label if checking multiple 
<img width="662" alt="Screenshot 2024-02-12 at 13 57 27" src="https://github.com/indico/indico/assets/54508387/301a1c9c-8221-4fa0-bf36-112c06d9321a">
This is the position if left the way it was and return True always
<img width="685" alt="Screenshot 2024-02-12 at 13 59 37" src="https://github.com/indico/indico/assets/54508387/5422b849-54ef-46b7-aaaa-1abaa51fc14f">